MechJeb2/VesselState.cs

```diff
@@ -151,6 +151,31 @@ public enum UllageState {
 
         public MovingAverage3d angularVelocityAvg = new MovingAverage3d(5);
 
+        // instantaneous values
+        public double currentPitch
+        {
+            get
+            {
+                return (rotationVesselSurface.eulerAngles.x > 180) ? (360.0 - rotationVesselSurface.eulerAngles.x) : -rotationVesselSurface.eulerAngles.x;
```

Comment:
MuUtils.Clamp180 for these?

Reply:
Ah I see where that already existed, but if you do more cleanup you might want to tidy that up.